apps/react-sdk/src/transpiler/transpiler.ts

```diff
@@ -16,7 +16,8 @@ const ROOT_DIR = Path.resolve(__dirname, '../..');
  * @param options any extra options that should be passed.
  */
 export async function transpileFiles(directory: string, outputDir: string, options?: TranspileFilesOptions) {
-    const { files, dirs } = await getStatsInDir(directory);
+    let { files, dirs } = await getStatsInDir(directory);
+    files = files.filter(f => !f.endsWith("asyncapi-ui.min.js"));
```

Comment:
`asyncapi-ui.min.js` is very html-template specific. Ideally we should not do changes like this here in sdk. 

best would be to suppress any warning by default and make them visible in the generator logs only if `debug` flag is passed